### PR TITLE
Sink credentials

### PR DESCRIPTION
**Pull request by clemensv**
AI "PR for credential properties"

### CODE REVIEW

File: subscriptions-api.md [hunk 1]
```diff
@@ -602,13 +617,90 @@ settings. All other settings SHOULD be supported.
 - Constraints:
   - REQUIRED
 
+#### 3.2.3 Sink Credentials
+
+A sink credential provides authentication or authorization information required
```
Comment (duglin) [inline subscriptions-api.md:622]: s/required/necessary/

File: subscriptions-api.md [hunk 2]
```diff
@@ -387,6 +387,21 @@ Each subscription is represented by an object that has the following properties:
...
   - `https://example.com/event-processor`
 
+##### sinkCredential
+- Type: Map of attributes
+- Description: A set of settings carrying credential information that 
+  is enabling the entity delivering events to the subscription target to 
+  be authorized for delivery at the `sink` endpoint. The well-known 
+  attribute values are defined in [section 3.2.3](#323-sink-credentials).
+
+  Implementations SHOULD NOT include secrets contained in this map when
```
Comment (duglin) [inline subscriptions-api.md:397]: why not "MUST NOT" ?

File: subscriptions-api.md [hunk 3]
```diff
@@ -387,6 +387,21 @@ Each subscription is represented by an object that has the following properties:
 - Examples:
   - `https://example.com/event-processor`
 
+##### sinkCredential
```
Comment (duglin) [inline subscriptions-api.md:390]: Do we need to say how these credentials are manifested on the wire?

**Comment by duglin** [comment]
@clemensv can you fix the RFC keywords that the bot is complaining about?

**Comment by duglin** [comment]
Approved on the 9/2 call - will merge when bot is happy

**Comment by duglin** [comment]
@clemensv just a reminder

File: subscriptions-api.md (hunk 2 above)
Comment (clemensv) [reply subscriptions-api.md:397]: Too parental.

File: subscriptions-api.md (hunk 3 above)
Comment (clemensv) [reply subscriptions-api.md:390]: No. Depends on the protocol and negotiations. Might be a SASL choice.

**Comment by duglin** [comment]
bot is happy, merging....

**Pull request merged**